data/tsv/mww_latn_phonemic.tsv

```diff
@@ -1,17 +1,12 @@
 cag	c a ˧˩̤
-caj dab	c a ˥˧ d a ˥
```

Comment:
this seems to be removing every multi-word expression in this file.

Reply:
Ah, we should have anticipated this. Any word with spaces in it (which will include a lot of Vietnamese entries) will get filtered out now.

(On a side note, are we supposed to be including words with spaces in them by default now? If not then the above entry in mww shouldn't be there and we need to rework our skip spaces logic.)